cranelift/codegen/src/verifier/mod.rs

```diff
@@ -1046,6 +1082,31 @@ impl<'a> Verifier<'a> {
         }
     }
 
+    fn verify_is_address(
+        &self,
+        loc_inst: Inst,
+        v: Value,
+        errors: &mut VerifierErrors,
+    ) -> VerifierStepResult {
+        if let Some(isa) = self.isa {
+            let pointer_width = isa.triple().pointer_width()?;
+            let value_type = &self.func.dfg.value_type(v);
```

Comment:
No need to borrow here -- it's a `Type` which wraps a u16 internally (ie, it's tiny and `Copy`).

Reply:
:+1: removed, definitely not necessary if it's Copy and fits in a register.